addon/mixins/infinity-custom.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 import InfinityRoute from 'ember-infinity/mixins/route';
 
-const assign = Ember.assign || Ember.merge;
```

Comment:
Maybe also note in the CHANGELOG that this PR remove usage of `Ember.merge` in favor of `Ember.assign`?